application-workloads/darktrace-vsensor-standalone/README.md

```diff
@@ -0,0 +1,338 @@
+
+The template should be used to deploy vSensor(s) in an existing Virtual Network (VNet) and an existing subnet.
+
+##### Resource Group
+The Resource Group (RG) is where the new resources’ metadata will be stored. A pre-existing RG may be selected for this purpose, and is an optional requirement. The template will create a RG if one is not provided.
+
+##### Network Security Group
+The template will create a new Network Security Group (NSG) that, amongst other rules, will enable access for management from the IP addresses and/or IP Ranges specified when deploying the template. 
+
+##### VM size and VM disk size
+The VM size should be decided based on the vSensor Requirements as described in the ‘Example Sizings’ in the below link:
+https://customerportal.darktrace.com/product-guides/main/vsensor-requirements
+
+When choosing the VM disk size the same ‘Example Sizings’ table can be used. Bear in mind that bigger disk size means larger PCAP storage.
+
+Note that, vSensor performance will vary by CPU speed and the nature of the traffic - estimated sizings are provided for guidance only.
+
+##### Admin User Public Key for ssh authentication
+A pre-existing pair private/public key for ssh access is required where the public key will be used by the template to configure the newly created user on the vSensor.
+
+##### Automated vSensor installation and Configuration
+For the automated vSensor installation please make sure that:
+*	The newly created VM has access to packages.darktrace.com and packages-cdn.darktrace.com on port 443/TCP
+*	You already have the Darktrace Update Key
+
+Follow the vSensor-FAQ link for more information regarding the above two requirements: 
+https://customerportal.darktrace.com/product-guides/main/vsensor-faq
+
+##### Communication between the vSensor and the Darktrace master instance (virtual/physical)
+To connect and register the vSensor to the Darktrace master instance the following is required:
+* The Darktrace master instance’s FQDN or IP and port
+* The push token (generated on the Darktrace master instance System Config page)
+* Access to the master instance’s FQDN or IP and port
+
+##### Pre-Configured osSensor HMAC
+The template will expect the osSensor HMAC Token to be provided so it can configure it as part of their automated configuration. 
+
+##### Access to the vSensor
+The vSensors created will not have public IP associated. Exposing a vSensor via a public IP carries additional security burden and is not a requirement of push token authentication with the master instance.
+
+It is expected that there is already a solution in place to access the VMs in the VNet via their private IP, for example via Azure Bastion Host.
+
+If there is a virtual firewall on your network, please ensure access is granted to the vSensor to the Darktrace instance FQDN/port or IP/port, as well as to packages.darktrace.com and packages-cdn.darktrace.com on port 443.
+
+
+
+How to deploy the template with Azure Portal
```

Comment:
remove this section, all the user has to do is click "Deploy to Azure" button above (and it's the same for all templates in the repo)

Reply:
the section was removed